plugins/modules/system/osx_defaults.py

```diff
@@ -76,47 +76,54 @@
 '''
 
 EXAMPLES = r'''
-- osx_defaults:
+- name: Manage macOS user defaults
```

Comment:
I'm not sure whether adding a generic name is better than having no name.

Reply:
maybe it's not better but the documentation says `each example should include a name: line`.
(and it's not worse)